test/cmd/authentication.sh

```diff
@@ -40,6 +40,10 @@ os::cmd::expect_success_and_text 'oc get projects --token="${allescalatingpowers
 
 accesstoken=$(oc process -f ${OS_ROOT}/test/fixtures/authentication/scoped-token-template.yaml TOKEN_PREFIX=access SCOPE=user:check-access USER_NAME="${username}" USER_UID="${useruid}" | oc create -f - -o name | awk -F/ '{print $2}')
 os::cmd::expect_success_and_text 'curl -k -XPOST -H "Content-Type: application/json" -H "Authorization: Bearer ${accesstoken}" ${API_SCHEME}://${API_HOST}:${API_PORT}/oapi/v1/namespaces/cmd-authentication/localsubjectaccessreviews -d @${OS_ROOT}/test/fixtures/authentication/localsubjectaccessreview.json' '"kind": "SubjectAccessReviewResponse"'
+os::cmd::expect_success_and_text 'oc policy can-i create pods --token=${accesstoken} -n cmd-authentication --ignore-scopes' 'yes'
```

Comment:
> Why would I use a scoped token and ignore its scope?

Contrived test case here, but the general reason is that its only token that I gave you.  You get a token that can only tell you what deads can do.  You then use that knowledge to allow indirect access to a resource on their behalf.  This allows a clever integration to use our authorization engine to ACL their own resources.